discovery-provider/src/queries/get_tracks.py

```diff
@@ -207,20 +215,22 @@ def _get_tracks(session, args):
     if "sort" in args and args.get("sort") is not None:
```

Comment:
Thanks for fixing these too. I think thye're mostly deprecated, but a great fix nonetheless

Reply:
actually the client seems to be using this and this is where we set the default